Porting/bisect-runner.pl

```diff
@@ -844,6 +844,29 @@ =head1 OPTIONS
 
 =back
 
+=head1 ABOUT BISECTION
+
+The process is all about identifying the commit that caused some change
+in behaviour - maybe good, maybe bad. But it is built around the
+C<git-bisect> tool, which is much more specifically oriented around
+finding "what broke the build". C<git> terminology embeds that assumption -
+commits earlier than the target commit are "good" commits, those later
```

Comment:
My impression is that we generally spell names of other programs with `F<>`, so I would recommend `F<git>` here, and `F<bisect.pl>` in two locations below.